versioned/tiered-tests/src/main/java/com/dremio/nessie/versioned/tests/AbstractTestStore.java

```diff
@@ -365,10 +363,17 @@ public void deleteSelectedByConditionExpression1NoAlias() {
    */
   @Test
   public void deleteSelectedByConditionExpression2NoAlias() {
-    ConditionExpression conditionExpression2 = ConditionExpression.of(ExpressionFunction.equals(ExpressionFunction.size(COMMITS), TWO));
+    final ConditionExpression conditionExpression2 = ConditionExpression.of(ExpressionFunction.equals(ExpressionFunction.size(COMMITS), TWO));
     deleteConditional(SampleEntities.createBranch(random), ValueType.REF, true, Optional.of(conditionExpression2));
   }
 
+  private <T extends HasId> void putWithCondition(T sample, ValueType type, boolean putSucceeded) {
```

Comment:
When is putSucceeded ever false?

Reply:
Answer: never. I handle the negative tests separately. Will remove.